source/Octopus.Tentacle/Kubernetes/Crypto/ScriptPodLogEncryptionKeyGenerator.cs

```diff
@@ -31,7 +31,7 @@ public async Task<byte[]> GenerateKey(ScriptTicket scriptTicket, int keySizeInBy
             var pdb = new Pkcs5S2ParametersGenerator(new Sha256Digest());
             
             //we use the machine encryption key as the password and the script ticket is the salt
-            pdb.Init(PbeParametersGenerator.Pkcs5PasswordToBytes(Encoding.ASCII.GetChars(machineEncryptionKey)), Encoding.UTF8.GetBytes(scriptTicket.TaskId), 1000);
+            pdb.Init(machineEncryptionKey, Encoding.UTF8.GetBytes(scriptTicket.TaskId), 1000);
```

Comment:
From the review in #1047, we are going through 2 conversions to get a byte array... when we have a byte array in the first place 😄